### ISSUE TITLE
Privet

### ISSUE THREAD

**Issue by vcka** [DokAndMax/android_device_lge_v10#1]
Maybe you have compiled flashable image?

**Comment by DokAndMax** [comment]
Да, но оно не запускается. TWRP работает на ядре, а прошивка нет.
И да, если что это для телефона LG X155.

**Comment by vcka** [comment]
A na LG V10 netu nechevo?

**Comment by DokAndMax** [comment]
Посмотри android_device_lge_v10-common, а если точнее https://github.com/LG-V10/android_device_lge_v10-common .

**Comment by vcka** [comment]
A eto dlia kotorogo androida?

**Comment by DokAndMax** [comment]
В репозитории ветка называется oreo , значит могу предположить что это Android 8
